policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/slic/base/child.yaml

```
@@ -0,0 +1,11 @@
+description: Saskatchewan low income tax credit base for each child.
```

Could you similarly revise the other parameter descriptions?
```suggestion
description: Saskatchewan provides this low income tax credit base amount for each child.
```